### PR TITLE
form created

### PR DESCRIPTION
**Pull request by kushalsourav**
signin signup created

### CODE REVIEW

**Review by Debarshi95 — APPROVED** [review]
Good to merge.

File: login.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                    <label for="username" class="input-label">password</label>
+                    <input type="text" placeholder="Enter your password" class="input">
+                </div>
+                <div class="form-control">
+                    <div class="form-info">
+                        <label>
+                            <input type="checkbox" 
+                                   checked="checked" name="remember">Remember me
+                          </label>
+                        <a href="#" class="btn btn-text">Forgot your password</a>
```
Comment (partha8) [inline login.html]: Reduce the size of forgot password.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    <div class="container">
+        <div class="form-box">
+            <div class="form form-primary">
+                <div class="form-control ">
+                    <label for="username" class="input-label">Username</label>
+                    <input type="text" placeholder="Enter your name" class="input">
+                </div>
+                <div class="form-control">
+                    <label for="username" class="input-label">password</label>
+                    <input type="text" placeholder="Enter your password" class="input">
```
Comment (partha8) [inline login.html]: While using input and label, you should have id attribute in input tag which is equal to the label's for attribute. For example, here for="username" in the label will match input's id="username". This is a good practice to incorporate. There are other places as well where you should make the changes.

File: signup.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                            <i class="fas fa-heart"></i></a>
+                    </li>
+                </ul>
+            </div>
+            <a href="login.html" class="btn btn-primary">Login</a>
+        </div>
+    </nav>
+    <div class="container">
+        <div class="form form-primary">
+            <div class="form-control ">
```
Comment (partha8) [inline signup.html:44]: Why didnt you use "form-box" class here as well? The signup form stretches the full width of the view width, having it in the center would be nice.

**Review by partha8 — APPROVED** [review]

File: signup.html (hunk 3 above)
Comment (kushalsourav) [reply signup.html:44]: if i use form-box here -- input widths will shrink thats why i made signup form width 100%

**Pull request merged**